### PR TITLE
[subject page] allow place groups for a topic page config

### PR DESCRIPTION
**Pull request by chejennifer**
- add list of allowed place groups in the subject page config
- allow equity page for all states in USA

Screenshot 2023-06-06 at 1 37 11 PM

bug fixes along the way (caused by updates in accepted parameter type and return value of parent_places):
- fix bug in getting parent place type in choropleth API
- fix bug in returned parent place list from subject page config place metadata

### CODE REVIEW

File: server/routes/topic_page/html.py [hunk 1]
```diff
@@ -96,6 +96,11 @@ def topic_page(topic_id=None, place_dcid=None):
 
   more_places = request.args.getlist('places')
 
+  # TODO: should use place metadata API to fetch these data in one call.
+  place_type = place_api.get_place_type(place_dcid)
```
Comment (shifucun) [inline server/routes/topic_page/html.py:100]: Can just use dc.get_place_info() to fetch metadata?

**Review by shifucun — APPROVED** [review]
very nice!

File: server/routes/topic_page/html.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/topic_page/html.py]: done

**Comment by chejennifer** [comment]
thanks for the review!

**Comment by pradh** [comment]
This is cool!

**Pull request merged**